fil_fungible_token/src/token/types.rs

```diff
@@ -98,6 +107,7 @@ pub trait FRC46Token<E> {
     fn burn_from(&mut self, params: BurnFromParams) -> Result<BurnFromReturn, E>;
 }
 
+type GranularityReturn = u64;
 type TotalSupplyReturn = TokenAmount;
 type BalanceReturn = TokenAmount;
 type AllowanceReturn = TokenAmount;
```

Comment:
Should these be made `pub` so that they can be used in actor code?

Reply:
They are aliases so you can mix and match them with the underlying type (compiler doesn't mind either way). I think it's worth exposing them so we can use in our example code, Though the underlying types also seem obvious enough for things like granularity or total supply.

Reply:
Yes please do.